src/test/java/de/mediathekview/mserver/crawler/rbb/parser/RbbFilmDetailDeserializerTest.java

```diff
@@ -65,10 +66,10 @@ public void deserializeTest() {
     final Document document = Jsoup.parse(htmlContent);
 
     final RbbFilmDetailDeserializer target = new RbbFilmDetailDeserializer();
-    final Optional<RbbFilmInfoDto> actual = target.deserialize(new CrawlerUrlDTO(requestUrl), document);
+    final Optional<FilmInfoDto> actual = target.deserialize(new CrawlerUrlDTO(requestUrl), document);
 
     assertThat(actual.isPresent(), equalTo(true));
-    RbbFilmInfoDto dto = actual.get();
+    FilmInfoDto dto = actual.get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "actual.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)